src/audio/mux/mux.c

```diff
@@ -71,7 +72,7 @@ static int mux_set_values(struct comp_data *cd, struct sof_mux_config *cfg)
 	return 0;
 }
 
-static struct comp_dev *mux_new(struct sof_ipc_comp *comp)
+UT_STATIC struct comp_dev *mux_new(struct sof_ipc_comp *comp)
```

Comment:
Do we need to pollute the code like this? We could just use the comp API and call this methods implicitly.

Reply:
This is unnecessary. It can be tested via public api.